src/SourceDialog/SourceDialog.vala

```diff
@@ -250,9 +251,19 @@ public class Maya.View.SourceDialog : Gtk.Grid {
             type_combobox.sensitive = false;
             type_combobox.set_active (0);
             list_store.foreach (tree_foreach);
-            is_default_check.active = false;
-            var cal = (E.SourceCalendar)source.get_extension (E.SOURCE_EXTENSION_CALENDAR);
 
+            try {
+                var registry = new E.SourceRegistry.sync (null);
+                var source_is_default = source.equal (registry.default_calendar);
+                debug (@"source_is_default: $source_is_default");
```

Comment:
Typically we use printf style:
```suggestion
                debug ("source_is_default: %b", source_is_default);
```

Reply:
Oops, that wasn't even supposed to be there. Deleted it.